bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -107,6 +110,8 @@ thing-type.shelly.shellypluswdus.description = Shelly Wall Dimmer US Device
 thing-type.shelly.shellyplus10v.description = Shelly Plus Dimmer 10V
 thing-type.shelly.shellyplusrgbwpm-color.description = Shelly Plus RGBW PM - RGBW Dimmer with Power Meter
 thing-type.shelly.shellyplusstrip.description = Shelly Plus Power Strip 4 
+thing-type.shelly.shellyplusem.description = Shelly Plus EM Power Meter
+thing-type.shelly.shellyplusem.description = Shelly Plus 3EM-63 - 3 Phase Power Meter, 63 Amp
```

Comment:
Something wrong here, this seems duplicate

Reply:
Good catch, I didn't notice that in #18925 where the code is directly copied from. I'll make the same comment there, and fix it in this branch.